trends_presentation/src/commonMain/kotlin/net/thechance/mena/trends/presentation/screen/home/component/ExpandableText.kt

```diff
@@ -0,0 +1,85 @@
+package net.thechance.mena.trends.presentation.screen.home.component
+
+import androidx.compose.animation.AnimatedContent
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.core.LinearOutSlowInEasing
+import androidx.compose.animation.core.tween
+import androidx.compose.animation.fadeIn
+import androidx.compose.animation.fadeOut
+import androidx.compose.animation.togetherWith
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import mena.trends_presentation.generated.resources.Res
+import mena.trends_presentation.generated.resources.show_less
+import mena.trends_presentation.generated.resources.show_more
+import net.thechance.mena.designsystem.presentation.component.text.Text
+import net.thechance.mena.designsystem.presentation.theme.theme.Theme
+import net.thechance.mena.trends.presentation.shared.component.modifier.noRippleClickable
+import org.jetbrains.compose.resources.stringResource
+
+@Composable
+fun ExpandableText(
+    text: String,
+    maxLines: Int = 2,
+    isExpanded: Boolean,
+    modifier: Modifier = Modifier,
+    onExpandedChange: (Boolean) -> Unit,
+    moreText: String = stringResource(Res.string.show_more),
+    lessText: String = stringResource(Res.string.show_less)
+) {
```

Comment:
we can make modifier as first optional parameter 
```suggestion 
@Composable
internal fun ExpandableText(
    text: String,
    maxLines: Int = 3,
    isExpanded: Boolean,
    onExpandedChange: (Boolean) -> Unit,
    modifier: Modifier = Modifier,
    moreText: String = stringResource(Res.string.show_more),
    lessText: String = stringResource(Res.string.show_less)
) {